kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala

```diff
@@ -185,6 +185,7 @@ trait TPCDSHelper {
       Field("cr_ship_mode_sk", int),
       Field("cr_warehouse_sk", int),
       Field("cr_reason_sk", int),
+      Field("cr_order_number", int),
```

Comment:
missed column

Reply:
Is the schemas different in different TPCDS versions? Or we just missed the column.

Reply:
just missed